README.md

```diff
@@ -7,7 +7,9 @@ Keras and XGBoost.
 
 API Examples
```

Comment:
Maybe call this just "API" or "API References" and then change the first link to "Example Code Snippets" ?

Reply:
Sure